src/FormField/ScopeBuilder.php

```diff
@@ -72,30 +72,49 @@ class ScopeBuilder extends Generic
      */
     protected $query = [];
 
+    protected const OPERATOR_EQUALS = 'equals';
+    protected const OPERATOR_DOESNOT_EQUAL = 'does not equal';
+    protected const OPERATOR_GREATER = 'is greater than';
+    protected const OPERATOR_GREATER_EQUAL = 'is greater or equal to';
+    protected const OPERATOR_LESS = 'is less than';
+    protected const OPERATOR_LESS_EQUAL = 'is less or equal to';
+    protected const OPERATOR_CONTAINS = 'contains';
+    protected const OPERATOR_DOESNOT_CONTAIN = 'does not contain';
+    protected const OPERATOR_BEGINS = 'begins with';
+    protected const OPERATOR_DOESNOT_BEGIN = 'does not begin with';
+    protected const OPERATOR_ENDS = 'begins with';
+    protected const OPERATOR_DOESNOT_END = 'does not begin with';
+    protected const OPERATOR_IN = 'is in';
+    protected const OPERATOR_NOT_IN = 'is not in';
+    protected const OPERATOR_MATCHES_REGEX = 'matches regular expression';
+    protected const OPERATOR_DOESNOT_MATCH_REGEX = 'does not match regular expression';
+    protected const OPERATOR_EMPTY = 'is empty';
+    protected const OPERATOR_NOT_EMPTY = 'is not empty';
+    
     /**
      * VueQueryBulder => Condition map of operators
      *
      * @var array
      */
     protected static $operators = [
-        'equals' => '=',
-        'does not equal' => '!=',
-        'is greater than' => '>',
-        'is greater or equal to' => '>=',
-        'is less than' => '<',
-        'is less or equal to' => '<=',
-        'contains' => 'LIKE',
-        'does not contain' => 'NOT LIKE',
-        'begins with' => 'LIKE',
-        'does not begin with' => 'NOT LIKE',
-        'ends with' => 'LIKE',
-        'does not end with' => 'NOT LIKE',
-        'is in' => 'IN',
-        'is not in' => 'NOT IN',
-        'matches regular expression' => 'REGEXP',
-        'does not match regular expression' => 'NOT REGEXP',
-        'is empty' => '=',
-        'is not empty' => '!=',
+        self::OPERATOR_EQUALS => '=',
+        self::OPERATOR_DOESNOT_EQUAL => '!=',
```

Comment:
Does `negative` operators matches nulls or is there any option to set it like so - we should have real inverse operator.

Reply:
Can you give an example as I do not fully understand what you mean?

Reply:
table cell with value `null` is not matched with `= 1` but also NOT matched with `!= 1`

Reply:
Now operators `empty`, `not empty` result in checking against `NULL` value

Reply:
![image](https://user-images.githubusercontent.com/2228672/88925942-8dc92e00-d275-11ea-819a-383012d93f73.png)

@georgehristov please check whole ui

then what feedback is left for this PR?

Reply:
Fixed, I believe it is RTM

I will use in in atk4/login and we'll see if any polishing necessary